config/locales/nl.yml

```diff
@@ -1,7 +1,5 @@
 nl:
   locomotive:
-    not_logged_in:
-      title: "Locomotive <small>Designed to save your time</small>"
```

Comment:
The localization workflow is to first add keys in english and then use the Transifex command line too to push those new keys to their platform. I assume all the translators get notified in order to fill them.
Finally, I use another Transifex command to pull the new versions in the project.